app/models/schema.rb

```diff
@@ -6,6 +6,7 @@ class Schema
 
   CONFIG_PATH = Rails.root.join("config/content_block_manager.yml").to_s
 
+  EMBED_CODE_VISIBLE_PROPERTY_KEY = "embed_code_visible".freeze
```

Comment:
I think this belongs in the previous commit

Reply:
Or possibly the first two commits are in the wrong order?

Reply:
thks, not sure how that happened!